help-center/faq/cloud-faq.md

```diff
@@ -0,0 +1,83 @@
+---
+id: cloud-faq
+title: Memgraph Cloud - Frequently asked questions
+sidebar_label: Memgraph Cloud
+slug: /faq/cloud
+---
+
+## Pricing and payment
+
+### What is the pricing?
+
+You can estimate the cost of Memgraph Cloud's service by selecting your cloud
+region and instance size with our [Cost
+Calculator](https://cloud.memgraph.com/pricing), or you can check them out at
+[Project rates](/docs/memgraph-cloud/payment#project-rates))
+
+### How can I use a coupon that I got for Memgraph Cloud?
+First, you need to add a credit card to your account. Then you can apply the
+coupon.
+
+### What will happen to my instance after the free trial?
```

Comment:
instance or project?

Reply:
Instance. You get a trial for instance, not a project.